report/slack.go

```diff
@@ -327,12 +327,30 @@ func attachmentText(vinfo models.VulnInfo, osFamily string, cweDict map[string]m
 			vinfo.Mitigations(osFamily)[0].Value)
 	}
 
-	return fmt.Sprintf("*%4.1f (%s)* %s %s\n%s\n```\n%s\n```%s\n%s\n",
+	certAlert := ""
+	var jpalert string
+	for _, ja := range vinfo.AlertDict.Ja {
```

Comment:
One CVE has multiple alerts.
Your code overwrites the alert, so your code can not show when One CVE has multiple alerts.

Reply:
ex. https://github.com/future-architect/vuls/blob/master/alert/cve_jp.go#L76